### PR TITLE
Adds tests for #1339

### PR DESCRIPTION
**Pull request by RobertoPrevato**

### CODE REVIEW

File: tests/test_server_proto.py [hunk 1]
```diff
@@ -685,6 +685,27 @@ async def test_server_proto_args_07(self):
             await self.con.fetchone(
                 'select schema::Object {name} filter .id=$id', id='asd')
 
+    async def test_server_proto_args_08(self):
```
Comment (elprans) [inline tests/test_server_proto.py:688]: I'd add a similar test for named args also

Comment (RobertoPrevato) [reply tests/test_server_proto.py:688]: You had a very good point: indeed, it fails with another kind of error with named arguments.

```
File "edgedb/protocol/protocol.pyx", line 998, in edgedb.protocol.protocol.SansIOProtocol.encode_args
RuntimeError: expected named arguments, got positional arguments
```

Comment (1st1) [reply tests/test_server_proto.py:688]: That RuntimeError should be changed to `errors.InterfaceError`

**Comment by 1st1** [comment]
Roberto, could you please rebase the PR onto most recent master?

File: tests/test_server_proto.py (hunk 1 above)
Comment (1st1) [reply tests/test_server_proto.py:688]: This PR is becoming stale, let's revive it @RobertoPrevato

**Comment by elprans** [comment]
Blocked on edgedb/edgedb-python#208

**Comment by 1st1** [comment]
@fmoor Frederick, can you take a look at this PR and see if we should close it or if we should rebase it?

**Comment by fmoor** [comment]
> see if we should close it or if we should rebase it?

I didn't find any equivalent tests in master.

**Comment by fmoor** [comment]
@1st1 all the clients handle this check before sending the wrong number of arguments to the server. Maybe we don't need tests here?

**Pull request closed without merging**